cypress/e2e/plugins/2-prototype-kit-plugin-tests/handle-plugin-update-when-a-dependency-is-now-required.cypress.js

```diff
@@ -10,7 +10,7 @@ const dependencyPluginName = 'GOV.UK Frontend'
 describe('Handle a plugin update', () => {
   after(restoreStarterFiles)
 
-  it('when a dependency is now required', () => {
+  it.skip('when a dependency is now required', () => {
```

Comment:
Is this one supposed to be skipped?  It looks like maybe it was meant to be temporary.

Reply:
This needs to either be deleted or revisited as the test no longer works as it did previously.  This is because the older version of common-templates does not specify in it's config that it depends on govuk-frontend but it does because the sass fails to compile leading to the sass compilation error and so we can no longer (at the moment) have a test where a plugin depends upon another plugin when it is updated.